### PR TITLE
PR test

### PR DESCRIPTION
**Pull request by AltTristan**
Test some changes here.

### CODE REVIEW

File: netcheck.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ COLOR_RESET="\033[0m"
 
 STRING_1="LINK RECONNECTED:                               "
 STRING_2="LINK DOWN:                                      "
-STRING_3="TOTAL DOWNTIME:                                 "
+STRING_3="TOTAL DOWNTIME IN SECONDS:                      "
```
Comment (AltTristan) [inline netcheck.sh:28]: I think this should show an ACK

**Review by AltTristan — COMMENTED** [review]
The review was conducted with joy on GitContext.com

File: netcheck.sh [hunk 2]
```diff
@@ -63,16 +63,6 @@ PRINT_MANAGESERVICE() {
   PRINT_HR
 }
 
-PRINT_INSTALL() {
```
Comment (TristanBrotherton) [inline netcheck.sh:66]: I don't think you should have removed this block of code.

File: netcheck.sh [hunk 3]
```diff
@@ -155,6 +145,26 @@ START_WEBSERVER() {
   esac
 }
 
+SETUP_SERVER() {
```
Comment (TristanBrotherton) [inline netcheck.sh:148]: What is the purpose of this function?

**Review by TristanBrotherton — CHANGES_REQUESTED** [review]
The review was conducted with joy on GitContext.com

**Review by AltTristan — COMMENTED** [review]
The review was conducted with joy on GitContext.com

File: netcheck.sh (hunk 1 above)
Comment (AltTristan) [inline netcheck.sh:28]: Now it has another comment.

**Review by AltTristan — COMMENTED** [review]
The review was conducted with joy on GitContext.com

**Pull request closed without merging**